dockernotes

```diff
@@ -1,5 +1,6 @@
 docker is virtual container which will host software in this container  
-
+ 
+docker is mostly created for developers by the developers
```

Comment:
AWESOME loved it 
if you have time pls provide with more docker compose notes 
and zalenium as well

Reply:
Alright i will definitely add